scripts/build.sh

```diff
@@ -1,56 +1,64 @@
 #!/bin/bash
 
 REPO_DIR="$( cd "$(dirname $( dirname "${BASH_SOURCE[0]}" ))" &> /dev/null && pwd )"
-BUILDER=''
-GO=''
+# flag for ci-only behavior (CI is auto-populated with 'true' in github actions)
+ensure_docker_write_permissions=$CI
 
 # default values
 skipcompilation=false
 
-while getopts d:s:b:g: flag
+while getopts d:s: flag
 do
     case "${flag}" in
-        d) distributions=${OPTARG};;
+        d) distribution=${OPTARG};;
         s) skipcompilation=${OPTARG};;
-        b) BUILDER=${OPTARG};;
-        g) GO=${OPTARG};;
     esac
 done
 
-[[ -n "$BUILDER" ]] || BUILDER='ocb'
-[[ -n "$GO" ]] || GO='go'
-
-if [[ -z $distributions ]]; then
-    echo "List of distributions to build not provided. Use '-d' to specify the names of the distributions to build. Ex.:"
-    echo "$0 -d otelcol"
+if [[ -z $distribution ]]; then
+    echo "Distribution to build not provided. Use '-d' to specify the name of the distribution to build. Ex.:"
+    echo "$0 -d nr-otel-collector"
     exit 1
 fi
 
 if [[ "$skipcompilation" = true ]]; then
     echo "Skipping the compilation, we'll only generate the sources."
 fi
 
-echo "Distributions to build: $distributions";
+echo "Distribution to build: $distribution";
 
-for distribution in $(echo "$distributions" | tr "," "\n")
```

Comment:
Is the thought that we'd move back to iterating on multiple distributions once we add the next one?

Reply:
Oh, i see it's handled in the root Makefile 👍